src/test/java/libs/InputValues.java

```diff
@@ -0,0 +1,46 @@
+package libs;
```

Comment:
What is the value of placing the test values / expected results in their own class?

Reply:
It's more on future re-usability. It's more convenient to have these available test inputs that can be use by any test classes/methods.